db/data/regions/DE.yml

```diff
@@ -29,6 +29,8 @@ format:
   show: "{firstName} {lastName}_{company}_{address1}_{address2}_{zip} {city}_{country}_{phone}"
 format_extended:
   edit: "{country}_{firstName}{lastName}_{company}_{streetName}{streetNumber}_{address2}_{zip}{city}_{phone}"
+address1_regex:
+- "^(?<streetName>[^\\d,]+[^\\d., ]) *([.,]{1,2})? *(?<streetNumber>\\d+(?: ?[a-z])?)$"
```

Comment:
It is quite common to put a period after the street name, sometimes without a space afterward. I also saw a couple instances of a period and a comma 🤷‍♀️

Reply:
Edit: this is because a period is often used to denote street name abbreviations, so it should not be interpreted as a delimiter and should be left in the street name